apps/src/blockly/addons/cdoUtils.ts

```diff
@@ -148,7 +148,10 @@ function loadHiddenDefinitionBlocksToWorkspace(
  */
 function prepareSourcesForWorkspaces(source: string, embedded: boolean) {
   const parsedSource = parseSource(source, embedded);
-  const procedureTypesToHide = [BLOCK_TYPES.behaviorDefinition];
+  const procedureTypesToHide = [];
+  if (!embedded) {
+    procedureTypesToHide.push(BLOCK_TYPES.behaviorDefinition);
```

Comment:
Previously, we were _always_ moving behavior definitions to a hidden workspace, as these are never seen on a student's main workspace. However, this made it impossible to embed these blocks anywhere.